lib/types/uuid.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * UUID type constructor
+ *
+ * #### Example:
+ *
+ *     const id = new mongoose.Types.UUID;
+ *
+ * @constructor UUID
+ */
+
+'use strict';
+
+const UUID = require('mongodb').BSON.UUID;
+const uuidSymbol = require('../helpers/symbols').UUIDSymbol;
+
+/**
+ * Getter for convenience with populate, see gh-6115
+ * @api private
+ */
+
+Object.defineProperty(UUID.prototype, '_uuid', {
```

Comment:
This property is unnecessary and incorrect. If we wanted to keep this property, it should be `_id`, because `_uuid` isn't as commonly used as `_id`. But, either way, not helpful.